consensus/istanbul/backend/announce.go

```diff
@@ -497,32 +523,66 @@ func (sb *Backend) handleAnnounceMsg(peer consensus.Peer, payload []byte) error
 		logger.Trace("Going to process an announce msg", "announce records", announceData.AnnounceRecords)
 		for _, announceRecord := range announceData.AnnounceRecords {
 			if announceRecord.DestAddress == sb.Address() {
```

Comment:
Nit: Please change this to `if announceRecord.DestAddress != sb.Address() { continue }` to reduce the indentation of this block.